docs/libcurl/libcurl-errors.3

```diff
@@ -184,8 +184,6 @@ This error code has been unified with CURLE_SSL_CACERT since 7.62.0. Its
 previous value was 51.
 .IP "CURLE_BAD_CONTENT_ENCODING (61)"
 Unrecognized transfer encoding.
-.IP "CURLE_LDAP_INVALID_URL (62)"
-Invalid LDAP URL.
```

Comment:
I think this is worth leaving in for a while since the web documentation is generated from it. I can imagine for some time there could be searches for CURLE_LDAP_INVALID_URL. Perhaps label it obsolete.

Reply:
Fair. I'll amend.